### PR TITLE
add tag instructions to README

### PR DESCRIPTION
**Pull request by jmtcsngr**
To prevent issues when making releases. Seems using Web UI is not compatible with our github actions for deploy.

### CODE REVIEW

**Comment by kjsanger** [comment]
You _could_ use the web UI to do the release by creating a tag. However, I'm not sure the tags are annotated in that case (and hence won't have author and date attached).

What you shouldn't do is create a _release_ by hand because that is done by `create-release.yml` and you'd end up with that automation failing. Create release exists so that we don't need to do it by hand, including building and uploading artifacts such as the Docker image.

**Comment by dkj** [comment]
> You _could_ use the web UI to do the release by creating a tag.

I _think_ the GitHub web UI only allows tag creation with a release - I don't see how to create a tag without a release in the web interface. If this is right I think the text is good to go...

File: README.md
Comment (kjsanger) [inline README.md:22]: git tag -a x.y.z -m x.y.z

No "release", please

File: README.md
Comment (kjsanger) [inline README.md]: Releases are created and deployed automatically by a GitHub Action on
tag creation. Using the web interface to create a release will cause that
automation to fail.

**Review by kjsanger — CHANGES_REQUESTED** [review]

File: README.md
Comment (kjsanger) [reply README.md:23]: Probably also best to `git checkout devel` again, to leave the checkout on a branch other than one that can do release automation

Comment (jmtcsngr) [reply README.md:23]: See your point. I was assuming all of this was done in a fresh clone but didn't mention it so my bad. A fast-forward merge devel -> master maybe(?) to complain if there are extra things in your local copy of master?

I think I agree with everything there but the last one. If you are working in your own clone, not a fresh one, `push --tags origin` will push all tags you may have locally. Including alphas and betas or whatever else you may have.

Comment (kjsanger) [reply README.md:23]: If you're working in a clone which has direct push access to the main repo, I'd say it's best not to be making any changes there that you don't want to push. Stuff like making extra tags, branches, changes should be done on a fork.

Comment (jmtcsngr) [reply README.md:23]: > If you're working in a clone which has direct push access to the main repo...

Don't dispute that. But people can and there is no way to stop them. Should we go for `Create a fresh clone of the repo`.

So it can be

```bash
git clone --branch master git@github.com:wtsi-npg/samtools_container.git samtools_container && cd $_
git tag -a 'x.y.z' -m 'release x.y.z'
git push origin x.y.z
```

Comment (kjsanger) [reply README.md:23]: I think that's fine as an example because you _do_ have to clone at least once.

Comment (jmtcsngr) [reply README.md:23]: Added an extra commit with the change. Would like to rebase and squash before merging, but don't want to drag the thing.

Comment (jmtcsngr) [reply README.md:23]: Would it be ok if I rebase and squash?

**Pull request merged**
